tools/run_tests/run_interop_tests.py

```diff
@@ -210,8 +209,10 @@ def global_env(self):
         return {}
 
     def unimplemented_test_cases(self):
-        # aspnetcore doesn't have a client so ignore all test cases.
-        return _TEST_CASES + _AUTH_TEST_CASES
+        return _SKIP_COMPRESSION + \
+            _SKIP_SPECIAL_STATUS_MESSAGE + \
+            _AUTH_TEST_CASES + \
+            ['cancel_after_first_response', 'ping_pong']
```

Comment:
This is temporary. When HttpClient streaming has been fixed these will be enabled.